test/pass.jsx

```diff
@@ -49,7 +49,7 @@ const SpreadProps = (props) => <div role="banner" {...props} />;
 class MyComponent extends Component {
   static propTypes = { message: PropTypes.string.isRequired };
 
-  constructor(props: PropTypes) {
```

Comment:
This doesn't match how React describes using PropTypes.

https://legacy.reactjs.org/docs/typechecking-with-proptypes.html

This is leftover Flow syntax, which we no longer support.

While Backpack is still mid-migration away from Flow `eslint-config-skyscanner` still works as in its `babel.config.js` (which `@babel/eslint-parser` picks up automatically) it uses `'@babel/preset-flow'`.

https://github.com/Skyscanner/backpack/blob/5af5770ad24430868817cdcdf12107da1949a793/babel.config.js#L38-L39